tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -133,7 +133,7 @@ internal SpanContext(ISpanContext parent, TraceContext traceContext, string serv
 
             if (parent is SpanContext spanContext)
             {
-                _rawTraceId = spanContext.RawTraceId ?? rawTraceId;
+                _rawTraceId = spanContext._rawTraceId ?? rawTraceId;
```

Comment:
I don't think we want to change this. `RawTraceId` generates the hex string and caches it in `_rawTraceId` if it is not set yet.

Reply:
Hmm, although it makes sense to not set `this._rawTraceId` yet until it's needed, so never mind.

Reply:
I was thinking on this, why we want to force the hexstring generation if the property does that on-demand already? we are also copying the TraceId instance, so it can do it on-demand as the parent.